### PR TITLE
Adds ban option for team antags

### PR DESCRIPTION
**Pull request by Jordie0608**

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
i like the branch name

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -934,6 +935,8 @@
 					joblist += jobPos
 			if("ghostroles")
 				joblist += list("pAI", "posibrain", "drone", "deathsquad", "lavaland")
+			if("teamantags")
+				joblist += list("operative", "revolutionary", "cultist", "servant of Ratvar", "abductor", "alien candidate")
```
Comment (lzimann) [inline code/modules/admin/topic.dm]: Maybe a conversion ban aswell, that would include revs, clock, cult and alien?

**Comment by BeeSting12** [comment]
Thank you

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (PKPenguin321) [reply code/modules/admin/topic.dm]: Yes do that too

**Pull request merged**